src/app/legacy/containers/ChartbeatAnalytics/utils/index.js

```diff
@@ -90,6 +119,9 @@ export const buildSections = ({
     Podcast: 'Podcasts',
   };
 
+  const primaryMediaType =
+    pageType === MEDIA_ARTICLE_PAGE ? getPrimaryMediaType(taggings) : type;
```

Comment:
Should we put this in the `switch` statement below? Will result in duplicating some logic, but I think thats ok to be a bit more explicit.